mage_integrations/mage_integrations/destinations/oracledb/utils.py

```diff
@@ -0,0 +1,111 @@
+from typing import Dict, List
+
+import singer
+
+from mage_integrations.destinations.sql.constants import SQL_RESERVED_WORDS
+from mage_integrations.destinations.utils import (
+    clean_column_name as clean_column_name_orig,
+)
+from mage_integrations.sources.constants import (
+    COLUMN_FORMAT_DATETIME,
+    COLUMN_TYPE_BOOLEAN,
+    COLUMN_TYPE_INTEGER,
+    COLUMN_TYPE_NULL,
+    COLUMN_TYPE_NUMBER,
+    COLUMN_TYPE_OBJECT,
+    COLUMN_TYPE_STRING,
+)
+
+LOGGER = singer.get_logger()
+
+
+def clean_column_name(col, lower_case: bool = True, handle_leading_underscore: bool = True):
```

Comment:
you should pass this arg to clean_column_name_orig as well.
also, if handle_leading_underscore is always false for oracledb, there's no need to add this optional arg to this method.

Reply:
In alter command, handle_leading_underscore is set to be false, the rest of the time it is set to be default value as true